apps/react-native/.coderc/env/env.prod.ts

```diff
@@ -1,12 +1,12 @@
 /// <reference path="../../src/types/app.d.ts" />
 
-import {ENV} from '@brandingbrand/kernel-app';
+import {ENV} from '@brandingbrand/code-app';
 
 const prod: ENV = {
   ios: {
-    name: 'kernel',
-    bundleId: 'com.kernel',
-    displayName: 'Kernel',
+    name: 'code',
+    bundleId: 'com.code',
+    displayName: 'Code',
```

Comment:
Anywhere we are using Code as a proper name let's use Flagship Code. `  displayName: 'Flagship Code',`

and other examples would be:  ` text: 'Flagship Code would like to utilize your camera',`